app/uploaders/validate_pdf.rb

```diff
@@ -22,6 +22,9 @@ def validate(temp_file)
   end
 
   def validate_pdf(file)
+    # Skip when the pdf file is generated by DisabilityCompensationFastTrackJob
+    return if file.respond_to?(:fast_track)
```

Comment:
I could just return nothing in that method or something of that sort too, since it's really just a check to see that the method exists on the file object at all (no other file object would possess that method besides ours). Up to you on what you prefer, I'm not strongly opinionated about it!